src/stdfblib/ita/OPCUA_MGR.cpp

```diff
@@ -1016,7 +1124,7 @@ void OPCUA_MGR::setMGMCommand(EMGMCommandType paCMD, CStringDictionary::TStringI
     mCommand.mSecondParam.pushBack(CStringDictionary::getInstance().insert(paSecondParam));
   }
   if (paAdditionalParams != nullptr) {
-    mCommand.mAdditionalParams.fromString(paAdditionalParams);
+    mCommand.mAdditionalParams = CIEC_STRING(paAdditionalParams);
```

Comment:
This will trigger deprecated warning "Use ctor with explicit length parameter".

Reply:
True, please add the length of the string. If it's guaranteed to be a nul terminated c-string you can use strlen

Reply:
I don't think this is worth the effort.  In the next PR planned by @m-meingast `mAdditionalParams` will be turned into a `std:string` as it should be.  

But as this is a bigger change independent of this one I suggested to him to leave this PR as is.

@MartinMelikMerkumians  is this ok for you?

Reply:
My question is how to enable OPC UA Device Management and Monitoring? Is it enabled by default?

Reply:
In 4diac IDE it will be available as soon as you build your 4diac FORTE with OPC UA. In 4diac IDE you need to set the device profile of your device to OPC UA xxx.

Reply:
@azoitl fine for me

Reply:
> In 4diac IDE it will be available as soon as you build your 4diac FORTE with OPC UA. In 4diac IDE you need to set the device profile of your device to OPC UA xxx.

Using milestone IDE here and no OPC UA device profile beside HOLOBLOC, FBDK2, and DynamicTypeLoad. I always build FORTE with OPC UA enabled.

Reply:
OPC UA deployment is currently only in the nightly build 4diac IDE. Later this week we will merge develop to freeze which will update the milestone 4diac IDE version.